spec/services/food_service_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe 'food service test' do
+  it 'can make a call to the api', :vcr do
+    json_response = FoodService.request_api("search?query=sweet potatoes")
+
+    expect(json_response).to be_a Hash
+    expect(json_response[:foodSearchCriteria][:query]).to eq('sweet potatoes')
+  end
+
+  it 'can search for sweet potatoes', :vcr do
+    json_response = FoodService.search("sweet potatoes")
+
+    expect(json_response).to be_a Hash
+    expect(json_response[:foodSearchCriteria][:query]).to eq('sweet potatoes')
+  end
```

Comment:
You can also test in here for the key's/values that you'd expect to get returned for your use later.